documentation/openapi.json

```diff
@@ -1725,15 +1734,13 @@
           "ActionExecuting",
           "Throttled",
           "Finished"
-        ],
-        "type": "string"
```

Comment:
Every example of an enum schema that I've run across in OpenAPI has the `type` property set to `string`. Could you add a schema transformer that fixes up enum schemas?